cmstestsuite/unit_tests/grading/tasktypes/OutputOnlyTest.py

```diff
@@ -29,8 +29,8 @@
     OUTCOME, TEXT, TaskTypeTestMixin
 
 
-FILE_001 = File(digest="digest of 001", filename="output_001.txt")
-FILE_023 = File(digest="digest of 023", filename="output_023.txt")
+FILE_001 = File(digest="digest of 001", filename="001.out")
```

Comment:
Sorry I really don't understand, what is the purpose of these changes?

Reply:
Because in the output only specifications, the output filename is "%s.out". however, the testcase is output_%s.txt.

This difference cause the test to fail.

Reply:
For documentation purposes: these changes are reverted in this PR since they are beyond the scope of this PR.